src/app/js/formats/bubbleChart/BubbleAdmin.js

```diff
@@ -50,20 +51,32 @@ class BubbleAdmin extends Component {
         args: defaultArgs,
     };
 
+    constructor(props) {
+        super(props);
+        this.handleColorsChange = this.handleColorsChange.bind(this);
+        this.state = {
+            colors:
+                this.props.args.colors != null
```

Comment:
I see that comparison very seldomly now.

I would have written

```js
this.props.args.colors || colorUtils.MULTICHROMATIC_DEFAULT_COLORSET,
```

Sure, there may be some cases where it does not work well, but which ones?

Reply:
Also, `colorUtils.MULTICHROMATIC_DEFAULT_COLORSET` should have the same value as `defaultArgs.colors`, so why not use the latter as a default value (advantage: only one location to define default value).

Reply:
Oh, wait. That's what you use line 91: `this.state.colors || defaultArgs.colors`.

Reply:
Oh yeah that is clearer !